docs/source/notebooks/bayesian_estimation_of_ode_parameters.ipynb

```diff
@@ -0,0 +1,596 @@
+{
```

Comment:
<p>In many places, PyMC3 uses the <code>test_value</code> stuff to make sanity checks. Can we support this also for ODEs?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3578/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Can you show me an example?